homeassistant/components/fritz/sensor.py

```diff
@@ -81,7 +83,7 @@ def __init__(
         self, fritzbox_tools: FritzBoxTools, device_friendlyname: str, sensor_type: str
     ) -> None:
         """Init FRITZ!Box connectivity class."""
-        self._sensor_data = SENSOR_DATA[sensor_type]
+        self._sensor_data: list[str | None | Callable] = SENSOR_DATA[sensor_type]
```

Comment:
`None` usually comes in the end.
```suggestion
        self._sensor_data: list[str | Callable | None] = SENSOR_DATA[sensor_type]
```